docs/dev/release.rst

```diff
@@ -188,20 +183,78 @@ You can create a clean checkout as follows (requires
      Other Changes and Additions
      ---------------------------
 
-#. Commit your changes of the, uh, change log::
+     A.C.1 (unreleased)
+     ==================
 
-     git add CHANGES.rst
-     git commit -m "Back to development: A.B.dev"
+     Bug Fixes
+     ---------
 
-#. Push out the updated code and tag. If applicable, change ``origin`` to point to
-   the remote that points to the repository being released::
+#. Commit your changes of the, uh, change log with a message, "Back to development: A.B.dev"
 
-     git push origin main
-     git push origin vX.Y.Z
+#. Follow procedures for :ref:`release-milestones`.
+
+#. For your own sanity unrelated to the release, grab the new tag for your fork::
+
+     git fetch upstream --tags
+
+Congratulations, you have just released a new version of Jdaviz!
+
+.. _release-bugfix:
+
+Releasing a bugfix version
+==========================
+
+The procedure for a bugfix release is a little different from a feature release - you will
+be cherry-picking bugfixes into an existing release branch, and will also need to do some
```

Comment:
I enabled auto-backport bot today, though it only works if there is no conflicts. The way we doing change log now, any PR with change log might not be possible to use this bot.

You can see the stuff that was (or attempted to be) backported by the bot: https://github.com/spacetelescope/jdaviz/pulls?q=is%3Apr+author%3Ameeseeksmachine